.gitattributes

```diff
@@ -6,3 +6,4 @@
 *.pkl filter=lfs diff=lfs merge=lfs -text
 *.tar.gz filter=lfs diff=lfs merge=lfs -text
 *.tgz filter=lfs diff=lfs merge=lfs -text
+vision/object_detection_segmentation/yolov2-coco/model/yolov2_coco.onnx filter=lfs diff=lfs merge=lfs -text
```

Comment:
It's not necessary to explicitly encode this file in .gitattributes.

Reply:
Thank you for your comment. I deleted it.